README.md

```diff
@@ -170,7 +170,7 @@ Steps:
   d) ArgoCD is set to point to your fork and the preview branch
   e) User is switched back to feature branch to create additional changes
 
-If you want to reset your environment you can run the `hack/util-update-app-of-apps.sh https://github.com/redhat-appstudio/infra-deployments.git staging main` to reset everything including your cluster to `https://github.com/redhat-appstudio/infra-deployments.git` and match the upstream config.
+If you want to reset your environment you can run the `hack/util-update-app-of-apps.sh https://github.com/redhat-appstudio/infra-deployments.git staging pre-kcp` to reset everything including your cluster to `https://github.com/redhat-appstudio/infra-deployments.git` and match the upstream config.
```

Comment:
Not here exactly, but can you add a paragraph about how to maintain changes to staging using the pre-kcp branch?

- pull the pre-kcp branch locally
- make your changes and commit them
- make a pull request to the pre-kcp branch and get those changes reviewed
- test the changes, either via CI on the new pull request, or by doing a local install and running the E2E tests
- use `kubectl apply -f argo-cd-apps/app-of-apps/all-applications-staging.yaml` to apply the changes to the staging cluster (or should we use the update-app-of-apps.sh)
- merge the changes

Reply:
I didn't see it as essential to mention it here. The "preview" section partially covers it, and it's a basic "Git & ArgoCD" knowledge after all. 
If you want to use `pre-kcp` AppStudio version, then you should use that branch for everything:
* running scripts
* making PRs
* updating your ArgoCD instance

I've been using the `kcp` branch in that way without any issues. 

But feel free to create a PR if you want to add that section anyway.

Reply:
Yeah, I'll create a PR.  It'll be against this branch.